jekyll/_docs/continuous-deployment-with-heroku.md

```diff
@@ -76,13 +75,25 @@ requests per hour per key.
 
 <h2 id="pre-or-post">Heroku with pre- or post-deployment steps</h2>
 
-If you'd like to run commands before or after deploying to Heroku, you'll have to use the 'normal' `deployment` syntax.
+If you'd like to run commands before or after deploying to Heroku, you'll have to use the 'normal' `deployment` syntax together with the Heroku CLI (pre-installed). This method does not pull in your Heroku creds that was set in the CircleCI UI. Instead, you'll need to set your Heroku API key via standard [private environment variables][doc-priv-envars]. In the following example, we set `HEROKU_EMAIL` and `HEROKU_PASSWORD` as private environment variables where `HEROKU_PASSWORD` is your Heroku API token.
```

Comment:
type: are you saying it should be "is", since.... the creds are technically still there?

I had used password because that's the field in netrc. Looking again, it seems to be more of a netrc standard then something Heroku pushed so I used token instead.